### PR TITLE
Added DrawLinkedZones()

### PR DESCRIPTION
**Pull request by edo9300**
This will draw a yellow pulsing recatngle, like the white one, when a zone is hovered, in the monster zones linked to a link monster
image

### CODE REVIEW

**Comment by DailyShana** [comment]
maybe blue is better?
consistent with the color of Link monsters

**Comment by edo9300** [comment]
@DailyShana i changed the color code with this blue 0261a2
image

**Comment by mercury233** [comment]
I prefer the yellow version.

**Comment by mercury233** [comment]
image
image
It don't work on opponent's zone for me.

File: gframe/drawing.cpp [hunk 1]
```diff
@@ -188,6 +193,36 @@ void Game::DrawBackGround() {
 		driver->drawVertexPrimitiveList(vertex, 4, matManager.iRectangle, 2);
 	}
 }
+void Game::DrawLinkedZones(ClientCard* pcard) {
+	int mark = pcard->link_marker;
+	matManager.mSelField.AmbientColor = 0xff0261a2;
+	driver->setMaterial(matManager.mSelField);
+	if (dField.hovered_sequence<5) {
+		if (mark & 0x10 && dField.hovered_sequence>0)
```
Comment (VanillaSalt) [inline gframe/drawing.cpp]: LINK_MARKER_LEFT = 0010 (octal)  not 0x10 (hexadecimal)

**Comment by edo9300** [comment]
@VanillaSalt so they are in octal? Sorry I misunderstood
@mercury233 I too think that yellow looks better

**Comment by Zayelion** [comment]
> maybe blue is better?

Stop arguing about nonsense both of you. Implement the theme system that was discussed earlier so this is a non-issue.

**Pull request merged**